app/controllers/users/sessions_controller.rb

```diff
@@ -31,10 +31,9 @@ def active
     end
 
     def timeout
-      if sign_out
-        analytics.track_event(Analytics::SESSION_TIMED_OUT)
-        flash[:timeout] = t('session_timedout')
-      end
+      analytics.track_event(Analytics::SESSION_TIMED_OUT)
+      sign_out
```

Comment:
Yeah, it may not matter, I was just trying to explain what I believe the old behavior does.

I think that the app will automatically sign out after a fixed period of time, so that might come up with two tabs open or something.

But not worth worrying about, LGTM also as-is